test/Main.purs

```diff
@@ -34,6 +34,12 @@ getSomethin db = J.queryDB db queryP params
     queryP = SProxy :: SProxy "select name, count from mytable where name = $name and count = $count"
     params = { "$name": "asdf", "$count": 4 }
 
+getAnother :: SL.DBConnection -> Aff Foreign
+getAnother db = J.queryDB db queryP params
+  where
+    queryP = SProxy :: SProxy """select name, count from mytable where name = $name and count = $count"""
```

Comment:
Could you change this test case to test the newline instance?

Reply:
whoop, fixed

Reply:
Cool, thanks